.github/workflows/cut-rc.yml

```diff
@@ -0,0 +1,547 @@
+name: Cut RC
+on:
+  milestone:
+    types:
+      - closed
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Full RC number (A.B.C-rc.D)
+        required: true
+        default: 0.0.0-rc.1
+        type: string
+      dry-run:
+        description: Dry-run
+        required: false
+        default: true
+        type: boolean
+
+env:
+  main_branch: master
+  docs_repository: openshift/openshift-docs
+  jira_projects: ROX, RS, RTOOLS
+  slack_channel: C03D78P4RFF
```

Comment:
How about a comment which channel this is?
Also, should we change this to something visible to others now?

Reply:
This is now the `epic-release-improvements` channel. We'll change it to `eng-release` eventually. Well, I could add that in a comment.